clicache/integration-test/NUnitTestAddTests.ps1

```diff
@@ -0,0 +1,67 @@
+﻿Param
```

Comment:
This is missing a license header (rat should be failing)

Reply:
You are correct. Let me see what is wrong with rat.

Reply:
Rat sees those ps1 files as binary. LOL

Probably some Windows line ending issues. I will try to fix it.

Reply:
hmm... my rat run didn't see them as binary https://github.com/moleske/geode-native/pull/37/checks?check_run_id=1964750486

Reply:
Visual Studio edit insists on UTF-8 with BOM and RAT sees that as binary. I fixed a few files.

Reply:
Rat on macOS sees them as text but on Linux, where we do the check, as binary. 🤷